docs/content/6.resources/3.security.md

```diff
@@ -1,12 +1,14 @@
 # Security
 
 This section mostly contains a list of possible security problems. Note that the below flaws exist with many libraries and frameworks we use in our day-to-day when building and working with APIs. Even your vanilla Nuxt app already posesses some of these shortcoming. Missing in the below list are estimates of how likely it is that one of the list-items may occur and what impact it will have on your app. This is because that heavily depends on:
-- your app: Are you building a fun project? A proof of concept? The next fort-nox money management app?
-- your environment: Building a freely available app for fun? Have authentication in front of your app and trust all users that successfully authenticated? Superb! Don't trust anyone? Then please be extra-careful when using this library and when building you backend in general
+
+-   your app: Are you building a fun project? A proof of concept? The next fort-nox money management app?
+-   your environment: Building a freely available app for fun? Have authentication in front of your app and trust all users that successfully authenticated? Superb! Don't trust anyone? Then please be extra-careful when using this library and when building you backend in general
```

Comment:
as above - don't do 2 spaces instead of 1 between the dash and the text please (:

Reply:
Also fixed!